config/tests/plugins/plugins_test.go

```diff
@@ -0,0 +1,98 @@
+/*
+Copyright 2023 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package plugins
+
+import (
+	"os"
+	"strings"
+	"testing"
+
+	"k8s.io/test-infra/prow/plugins"
+	"sigs.k8s.io/yaml"
+)
+
+// TestApprovePluginConfig validates that there are no duplicate repos in the approve plugin config.
+func TestApprovePluginConfig(t *testing.T) {
```

Comment:
The validation in both of these functions is not specific to K8s-Prow. We should run this validation more generally, either as part of loading the config or in check-config, I think either location would be fine.

Reply:
Do you suggest keep the tests in Prow instead of moving them out? I thought we don't want move `config/prow/plugins.yaml` file.

Reply:
I just saw this was still open.
We don't want to move `config/prow/plugins.yaml` that is correct. I'm suggesting moving this validation to the check-config tool since it is not specific to this repo or Prow instance and should really be part of the generic config validation. The move shouldn't require much more than a copy+paste.